src/librbd/operation/DisableFeaturesRequest.h

```diff
@@ -88,13 +88,14 @@ class DisableFeaturesRequest : public Request<ImageCtxT> {
    *    v
    * STATE_UPDATE_FLAGS
    *    |
+   *    | (unblock exclusive lock)
    *    v
    * STATE_NOTIFY_UPDATE
    *    |
    *    v
    * STATE_RELEASE_EXCLUSIVE_LOCK (skip if not
    *    |                          required)
-   *    | (unblock writes)
+   *    | (unblock exclusive lock, skip if not required)
```

Comment:
I don't understand this change. I though just moving "(unblock writes)" before "STATE_NOTIFY_UPDATE" would be enough.

Reply:
This was in the error handling part which was in handle_finish(). But sure, if its not useful I can drop it.